### PR TITLE
Add release notes for Istio 1.2.3

### PR DESCRIPTION
**Pull request by fpesce**
- Also introduce a cosmetic style fix on cve-2019-12995 that was missing
  proper new line before 2 lists.

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[X] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/boilerplates/notes/1.2.3.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+- Fix infinite loops self-requests when pod defines a port undefined for service ([Issue 14443](https://github.com/istio/istio/issues/14443)) and ([Issue 14242](https://github.com/istio/istio/issues/14242))
+- Fix a bug where Stackdriver adapter is getting shutdown after telemetry is started.
+- Fix Redis Connectivity Issues.
+- Fix case-sensitivity in regex-based HTTP URI matching for Virtual Service ([Issue 14983](https://github.com/istio/istio/issues/14983))
+- Fix HPA and CPU settings for demo profile ([Issue 15338](https://github.com/istio/istio/issues/15338))
+- Relax Keep-Alive enforcement policy to avoid dropping connections under load ([Issue 15088](https://github.com/istio/istio/issues/15088))
+- When SDS is not used, skip Kubernetes JWT authentication to mitigate the risk of compromised (untrustworthy) JWTs being used.
+
+## Tests upgrade
+- Update base image version for Bookinfo reviews sample app ([Issue 15477](https://github.com/istio/istio/issues/15477))
```
Comment (geeknoid) [inline content/boilerplates/notes/1.2.3.md]: Please insert a blank line after the title.

File: content/boilerplates/notes/1.2.3.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+## Bug fixes
+
+- Fix infinite loops self-requests when pod defines a port undefined for service ([Issue 14443](https://github.com/istio/istio/issues/14443)) and ([Issue 14242](https://github.com/istio/istio/issues/14242))
+- Fix a bug where Stackdriver adapter is getting shutdown after telemetry is started.
+- Fix Redis Connectivity Issues.
+- Fix case-sensitivity in regex-based HTTP URI matching for Virtual Service ([Issue 14983](https://github.com/istio/istio/issues/14983))
+- Fix HPA and CPU settings for demo profile ([Issue 15338](https://github.com/istio/istio/issues/15338))
+- Relax Keep-Alive enforcement policy to avoid dropping connections under load ([Issue 15088](https://github.com/istio/istio/issues/15088))
+- When SDS is not used, skip Kubernetes JWT authentication to mitigate the risk of compromised (untrustworthy) JWTs being used.
+- Update base image version for Bookinfo reviews sample app ([Issue 15477](https://github.com/istio/istio/issues/15477))
```
Comment (fpesce) [inline content/boilerplates/notes/1.2.3.md]:
```suggestion

## Tests upgrade

- Update base image version for Bookinfo reviews sample app ([Issue 15477](https://github.com/istio/istio/issues/15477))
```

**Comment by geeknoid** [comment]
Is this ready to be published? If so, I'll hit the green button...

**Comment by fpesce** [comment]
@geeknoid I think I need to un-draft the istio/istio/release, I'll ping you in a short while
@duderino What do you think of it? Ready to go?

**Comment by duderino** [comment]
@fpesce I think you're getting the hang of this

/lgtmj

File: content/boilerplates/notes/1.2.3.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+## Bug fixes
+
+- Fix infinite loops self-requests when pod defines a port undefined for service ([Issue 14443](https://github.com/istio/istio/issues/14443)) and ([Issue 14242](https://github.com/istio/istio/issues/14242))
```
Comment (duderino) [inline content/boilerplates/notes/1.2.3.md]:
```suggestion
- Fix a bug where the sidecar could infinitely forward requests to itself when pod defines a port undefined for service ([Issue 14443](https://github.com/istio/istio/issues/14443)) and ([Issue 14242](https://github.com/istio/istio/issues/14242))
```

File: content/boilerplates/notes/1.2.3.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+- Fix a bug where Stackdriver adapter is getting shutdown after telemetry is started.
```
Comment (duderino) [inline content/boilerplates/notes/1.2.3.md]:
```suggestion
- Fix a bug where Stackdriver adapter shutsdown after telemetry is started.
```

File: content/boilerplates/notes/1.2.3.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+- Fix Redis Connectivity Issues.
```
Comment (duderino) [inline content/boilerplates/notes/1.2.3.md]:
```suggestion
- Fix Redis connectivity issues.
```

File: content/boilerplates/notes/1.2.3.md [hunk 6]
```diff
@@ -0,0 +1,14 @@
+## Bug fixes
+
+- Fix a bug where the sidecar could infinitely forward requests to itself when pod defines a port undefined for service ([Issue 14443](https://github.com/istio/istio/issues/14443)) and ([Issue 14242](https://github.com/istio/istio/issues/14242))
+- Fix a bug where Stackdriver adapter shutsdown after telemetry is started.
```
Comment (fpesce) [inline content/boilerplates/notes/1.2.3.md]:
```suggestion
- Fix a bug where Stackdriver adapter shuts down after telemetry is started.
```

**Comment by fpesce** [comment]
I guess it's ready to be published @geeknoid

**Pull request merged**